pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter.go

```diff
@@ -117,17 +128,73 @@ func validateRequestResource(ar *admissionv1.AdmissionReview, resourceType strin
 	return nil
 }
 
-func validateDedicatedCPUPlacement(instancetypeSpec *instancetypev1alpha2.VirtualMachineInstancetypeSpec) []metav1.StatusCause {
-	if instancetypeSpec == nil {
-		return nil
+func validateInstanceTypeCPU(instancetypeSpec *instancetypev1alpha2.VirtualMachineInstancetypeSpec) []metav1.StatusCause {
+	if instancetypeSpec != nil {
```

Comment:
Can you switch this back to `if instancetypeSpec == nil {` to avoid indenting all of the logic below?